2022-04-14 17:26:13,665 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-04-14 17:26:13,665 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/instabot/bot/bot.py
2022-04-14 17:26:13,666 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-04-14 17:26:14,149 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-04-14 17:26:14,556 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-04-14 17:26:14,966 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-04-14 17:26:15,323 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-04-14 17:26:18,242 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-04-14 17:26:18,243 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEQOceUu6kcLxEd9ZPQMzGYkmLoeHNIyv0itEf8xwCQ9xVaMbNkrz7rVfSjYeQDaDC0/ZMAXSRBKgirsvTfN/","api_path":"/challenge/AXEQOceUu6kcLxEd9ZPQMzGYkmLoeHNIyv0itEf8xwCQ9xVaMbNkrz7rVfSjYeQDaDC0/ZMAXSRBKgirsvTfN/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-04-14 17:26:18,243 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-04-14 17:26:18,243 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-04-14 17:26:18,243 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
